pkgs/by-name/gi/gitSetupHook/gitSetupHook.sh

```diff
@@ -0,0 +1,9 @@
+# shellcheck shell=bash
+# This script is a hook that sets up a minimal git configuration
+
+gitSetup () {
+    @gitMinimal@ config --global user.name GitSetupHook
+    @gitMinimal@ config --global user.email gitSetupHook@example.com
```

Comment:
According to https://www.rfc-editor.org/rfc/rfc2606 we could also use something like `GitSetupHook@nixpkgs.invalid` as the domain, `.invalid` being reserved for these things.